Firestore/Source/Remote/FSTStream.mm

```diff
@@ -258,13 +259,13 @@ - (void)startWithDelegate:(id)delegate {
   FSTAssert(_delegate == nil, @"Delegate must be nil");
   _delegate = delegate;
 
-  [self.credentials getTokenForcingRefresh:NO
-                                completion:^(Token result, NSError *_Nullable error) {
-                                  error = [FSTDatastore firestoreErrorForError:error];
-                                  [self.workerDispatchQueue dispatchAsyncAllowingSameQueue:^{
-                                    [self resumeStartWithToken:result error:error];
-                                  }];
-                                }];
+  _credentials->GetToken(
+      false, [self](Token result, const int64_t error_code, const absl::string_view error_msg) {
```

Comment:
(See above.) This time, the `false` isn't document, so I really have no idea what it means. Either document the false or create a constant (preferred.)